### PR TITLE
fix: syntax highlighting for prisma code

### LINKED ISSUE #2476: Issue with Prisma syntax highlighting
For example in this https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations#mongodb:

image

The fields and their types are alternating in violet and black.

### PR DESCRIPTION
**Pull request by nilubava**
## What issue does this fix?

Fixes #2476

### CODE REVIEW

**Comment by janpio** [comment]
While this seems to fix some problems, it also creates some new ones. Not sure if that is intentional or wanted:
image
Note how the relation field names are now formatted like types.

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx [hunk 1]
```diff
@@ -445,6 +445,14 @@ model Category {
   name    String
   postIDs String[] @db.ObjectId
   posts   Post[]   @relation(fields: [postIDs], references: [id])
+  Post    Post[]
+}
+
+enum Sample {
```
Comment (keerlu) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]: Something odd's been added here

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx [hunk 2]
```diff
@@ -445,6 +445,14 @@ model Category {
...
+  GOOD
+  Bad
+  Xo
+  Cy
 }
```
Comment (nikolasburk) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]:
```suggestion
```

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx (hunk 1 above)
Comment (nikolasburk) [reply content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]: I _think_ I fixed this with my [suggestion](https://github.com/prisma/docs/pull/3611#pullrequestreview-1105838055)

Comment (keerlu) [reply content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]: Looks like there's still a deleted `}`?

**Comment by keerlu** [comment]
Issue with the alternating fields is now fixed, happy with this once the accidental docs edit is reverted (edit: done)

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx [hunk 3]
```diff
@@ -445,7 +445,6 @@ model Category {
   name    String
   postIDs String[] @db.ObjectId
   posts   Post[]   @relation(fields: [postIDs], references: [id])
-}
 ```
```
Comment (nikolasburk) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]:
```suggestion
}
```
```

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx [hunk 4]
```diff
@@ -446,7 +446,6 @@ model Category {
   postIDs String[] @db.ObjectId
   posts   Post[]   @relation(fields: [postIDs], references: [id])
 }
```
Comment (nikolasburk) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx:448]:
```suggestion
}
```
```

Comment (nikolasburk) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]:
~~~suggestion
}
```
~~~

**Review by keerlu — APPROVED** [review]

**Comment by nikolasburk** [comment]
Ah thanks for catching this @keerlu , should be better now!

**Review by nikolasburk — CHANGES_REQUESTED** [review]
Looks good to me 👍

**Review by nikolasburk — APPROVED** [review]

**Pull request merged**